sql/backup.sh

```diff
@@ -142,6 +141,20 @@ pg_rman show detail -B ${BACKUP_PATH} > ${TEST_BASE}/TEST-0004.log 2>&1
 grep -c OK ${TEST_BASE}/TEST-0004.log
 
 echo '###### BACKUP COMMAND TEST-0005 ######'
+echo '###### Make sure incremental backup work ######'
```

Comment:
Is the following better?

Make sure that pg_rman doesn't take a differential backup, but a incremental backup